lego-webapp/app/routes/hiddenAdmin/components/BannerOverview.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+  Card,
+  ConfirmModal,
+  Icon,
+  LinkButton,
+  Page,
+} from '@webkom/lego-bricks';
+import { usePreparedEffect } from '@webkom/react-prepare';
+import { Trash2 } from 'lucide-react';
+import { ContentSection, ContentMain } from '~/components/Content';
+import { deleteBanner, fetchAllBanners } from '~/redux/actions/BannerActions';
+import { useAppDispatch, useAppSelector } from '~/redux/hooks';
+import { selectAllBanners } from '~/redux/slices/banner';
+
+const BannerOverview = () => {
+  const dispatch = useAppDispatch();
+  usePreparedEffect('fetchAllBanners', () => dispatch(fetchAllBanners()), []);
+  const allBanners = useAppSelector((state) => selectAllBanners(state));
+  return (
+    <Page
+      title={'Banner List'}
+      actionButtons={
+        <LinkButton href={`/sudo/banners/create/`}>Lag ny</LinkButton>
+      }
+    >
+      <ContentSection>
+        <ContentMain>
+          {allBanners.map((b) => (
+            <Card key={b.id}>
+              <h2>Tittel: {b.header}</h2>
+              <h3>Undertittel: {b.subheader}</h3>
+              <p>Lenke: {b.link}</p>
+              <LinkButton href={`/sudo/banners/edit/${b.id}/`}>
+                Rediger
+              </LinkButton>
+              <ConfirmModal
+                title="Bekreft Sletting av banner"
+                message={`Er du sikker på at du vil slette dette banneret?`}
```

Comment:
```suggestion
                title="Bekreft sletting av banner"
                message="Er du sikker på at du vil slette dette banneret?"
```
Is it "banneret" or "banneren"?

Reply:
[banneret!](https://ordbokene.no/nno/bm/banner)